dynamic_grammar.py

```diff
@@ -0,0 +1,22 @@
+from pythainlp.parse import dependency_parsing

```

Comment:
This requires downloading a ~3GB neural model; is that ideal? Will that work with your web app, and if so, is there a way to put it into the repo? Maybe you can quantize it after testing it on a number of sentences to be able to compare the output.